lib/values/app_theme.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+
+import 'app_colors.dart';
+
+class AppTheme {
+  AppTheme._();
+
+  // Fonts
+  static const ubuntuFont = 'Ubuntu';
+  static const nautigalFont = 'TheNautigal';
+
+  //To display medium fonts
+  static const TextStyle displayMedium = TextStyle(
+    color: AppColors.lightYellow,
+    fontSize: 16,
+    fontFamily: ubuntuFont,
+  );
+
+  //To display heading
+  static const TextStyle displayHeading = TextStyle(
+      fontSize: 80,
+      color: AppColors.welcomeColor,
+      fontWeight: FontWeight.bold,
+      fontFamily: AppTheme.nautigalFont);
+
+  //To display sub heading
+  static const TextStyle displaySubHeading = TextStyle(
+      fontSize: 30,
+      color: AppColors.lightYellow,
+      fontFamily: AppTheme.ubuntuFont,
+      fontWeight: FontWeight.bold);
+
+  static ThemeData appThemeLight() {
+    return ThemeData(
+      primaryColor: AppColors.blueGrey,
+      scaffoldBackgroundColor: AppColors.white,
+      textTheme: const TextTheme(
+        titleLarge: TextStyle(color: AppColors.black), // AppBar text color
```

Comment:
Please add comma for better readability.

Reply:
Resolved